setup.py

```diff
@@ -87,7 +87,7 @@ def run(self):
 
 
 requirements = [
-    "galaxy-importer==0.4.2",
+    "galaxy-importer==0.4.4",
```

Comment:
this edit also needs to be also done in the template_config https://github.com/ansible/galaxy_ng/blob/master/template_config.yml#L16

then you need to run the plugin template script, directions are in the last bullet here https://github.com/ansible/galaxy_ng/wiki/Updating-Dependencies